test/tools/router_check/router_check.cc

```diff
@@ -0,0 +1,9 @@
+// NOLINT(namespace-envoy)
```

Comment:
Lets add a comment or a file in this folder to explain why we have this stub (ie to be a stub for ci)